src/m1_facilitator/Config/Config.ts

```diff
@@ -0,0 +1,189 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import fs from 'fs-extra';
+import yaml from 'js-yaml';
+
+/**
+ * The class holds database configurations.
+ */
+export class DBConfig {
+  /** Database path */
+  public path: string;
+
+  public constructor() {
+    this.path = '';
+  }
+}
+
+/**
+ * It holds chain information like .
+ */
+export class Chain {
+  /** Chain rpc endpoint. */
+  public readonly nodeRpcEndpoint: string;
```

Comment:
Why RPC?

Reply:
Updated to `nodeEndpoint`. It's aligned with manifest.